lib/routes/detail/widgets/image_viewer.dart

```diff
@@ -0,0 +1,43 @@
+import 'package:flutter/material.dart';
+
+class ImageViewer extends StatefulWidget {
+  final int beerId;
+  final String imageURL;
+
+  ImageViewer({@required this.beerId, @required this.imageURL})
+      : assert(beerId != null),
+        assert(imageURL != null);
+  @override
+  _ImageViewerState createState() => _ImageViewerState();
+}
+
+class _ImageViewerState extends State<ImageViewer>
+    with SingleTickerProviderStateMixin {
+  double aspectRatio = 1;
+  @override
+  Widget build(BuildContext context) {
+    return TweenAnimationBuilder<double>(
+        tween: Tween<double>(begin: aspectRatio, end: aspectRatio),
+        duration: const Duration(seconds: 1),
+        builder: (BuildContext context, double size, Widget child) {
+          return AspectRatio(
+            aspectRatio: size,
+            child: InkWell(
+                child: Image.network(widget.imageURL),
+                onTap: () {
+                  setState(() {
+                    aspectRatio == 1 ? aspectRatio = 0.52 : aspectRatio = 1;
+                  });
+                }),
+          );
+        });
+  }
+}
+
+/*AspectRatio(
+aspectRatio: 16 / 9,
+child: Hero(
+tag: beer.id,
+child: Image.network(beer.imageURL),
+),
+))*/
```

Comment:
to remove

Reply:
Par contre, il faut garder le Composant Hero, c'est l'animation qui permet de d'avoir l'image qui se déplace entre la liste et le détail.

Reply:
yep